### PR TITLE
Optimize the document by reformat table

### PR DESCRIPTION
**Pull request by Fish-pro**
Signed-off-by: Fish-pro <zechun.chen@daocloud.io>

Optimize the document by reformat table and fix item numbering

The following pictures show the modified content for the convenience of review
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Kubeadm's scope is limited to the local node filesystem and the Kubernetes API,
 
 ## Common Kubeadm cmdlets
 1. **kubeadm init** to bootstrap the initial Kubernetes control-plane node.
-1. **kubeadm join** to bootstrap a Kubernetes worker node or an additional control plane node, and join it to the cluster.
-1. **kubeadm upgrade** to upgrade a Kubernetes cluster to a newer version.
-1. **kubeadm reset** to revert any changes made to this host by kubeadm init or kubeadm join.
+2. **kubeadm join** to bootstrap a Kubernetes worker node or an additional control plane node, and join it to the cluster.
+3. **kubeadm upgrade** to upgrade a Kubernetes cluster to a newer version.
+4. **kubeadm reset** to revert any changes made to this host by kubeadm init or kubeadm join.
```
Comment (SataQiu) [inline README.md:19]: This is not necessary, markdown will do it automatically, and hard coding here will increase our maintenance cost.
See https://github.com/kubernetes/kubeadm#common-kubeadm-cmdlets

Comment (Fish-pro) [reply README.md:19]: Thanks, I got it

**Review by SataQiu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**